### PR TITLE
Python Package Release

### PR DESCRIPTION
**Pull request by ngparas**
This PR includes the python package noted in #83 

It does have some integration testing that can be run locally, but that will likely require some changes to work with travis.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,12 @@ devtools::install_github("UptakeOpenSource/uptasticsearch")
 
 ### Python <a name="pythoninstallation"></a>
 
-We plan to release a Python implementation of this functionality, but that is not available at this time. Check back often!
+To install the python package:
```
Comment (jameslamb) [inline README.md]: Can you add language about PyPi to be explicit? Like

*This package is not currently available on PyPi. To build the development version of the package from source, clone the repo and then do the following:*

File: py-pkg/dummy_data/es_5_mapping.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (jameslamb) [inline py-pkg/dummy_data/es_5_mapping.json]: totally cool to check all of these in now. Can you please create an issue (you can do it before we've even merged this, honestly) indicating that we need to unify the CI strategies for R and Python?

Here's an example (if you want to link it) of testing R and Python packages in different sub-builds within one travis build using their nifty `matrix` thing: https://github.com/dmlc/xgboost/blob/master/.travis.yml#L15

File: py-pkg/dummy_data/load_dummy_data.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#! /bin/sh
```
Comment (jameslamb) [inline py-pkg/dummy_data/load_dummy_data.sh]: in the issue about CI, can you explicitly call out all the files currently under `py-pkg` which are redundant with what's currently available for CI in this repo? Basically so whoever picks up the issue knows what stuff to go clean up.

File: py-pkg/tests/test_util.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        """raises ValueError w/o proper port
+        """
+        with pytest.raises(ValueError):
+            _format_es_url("es.cluster.com/")
+        with pytest.raises(ValueError):
+            _format_es_url("es.cluster.com")
+        assert isinstance(_format_es_url("http://es.com:9200"), str)
+        assert isinstance(_format_es_url("es.es.com:9200"), str)
+
+    def test_nonsense(self):
```
Comment (jameslamb) [inline py-pkg/tests/test_util.py]: best tests

File: py-pkg/uptasticsearch/aggs_parsers.py [hunk 5]
```diff
@@ -0,0 +1,171 @@
+
+import pandas as pd
+
+
+def _terms_agg_to_df(aggs_json):
```
Comment (jameslamb) [inline py-pkg/uptasticsearch/aggs_parsers.py]: can you please create another issue that talks about how the "supported aggs" table in the README needs to get updated to talk about Python?

File: py-pkg/Makefile [hunk 6]
```diff
@@ -0,0 +1,20 @@
+
+install:
+	# Install uptasticsearch Python package
+	cd py-pkg && \
```
Comment (jameslamb) [inline py-pkg/Makefile]: can you add one more issue around moving R into `r-pkg/` for consistency? That would include updating any documentation, Travis, etc. so definitely should be its own PR

File: py-pkg/dummy_data/es_5_mapping.json (hunk 2 above)
Comment (ngparas) [reply py-pkg/dummy_data/es_5_mapping.json]: #85

File: py-pkg/dummy_data/load_dummy_data.sh (hunk 3 above)
Comment (ngparas) [reply py-pkg/dummy_data/load_dummy_data.sh]: updated #85

File: py-pkg/uptasticsearch/aggs_parsers.py (hunk 5 above)
Comment (ngparas) [reply py-pkg/uptasticsearch/aggs_parsers.py]: #86

File: py-pkg/Makefile (hunk 6 above)
Comment (ngparas) [reply py-pkg/Makefile]: #87

**Comment by jameslamb** [comment]
woo! Thanks @ngparas !
